sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+friendly_name: Newtab Interactions
+description: |-
+  Interactions on the newtab page, roughly one row per newtab "visit" per-day
+  (identified by the visit id).
+owners:
+- anicholson@mozilla.com
+labels:
+  application: firefox
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_newtab
+bigquery:
+  time_partitioning:
+    field: submission_date
+    type: day
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields:
+    - channel
+    - country
+references: {}
```

Comment:
Should `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1` be added to references? I'm not actually quite sure how that key works...